projects/step-core/src/lib/client/augmented/services/augmented-scheduler-service.ts

```diff
@@ -26,4 +28,8 @@ export class AugmentedSchedulerService extends SchedulerService {
     //@ts-ignore
     return this._httpClient.post<any>('rest/scheduler/task/' + id + '/execute', null, { responseType: 'text' });
   }
+
+  public isSchedulerEnabled(): Observable<boolean> {
+    return this._httpClient.get<any>('rest/settings/scheduler_enabled');
```

Comment:
If you know that result will be boolean, it make sense to write invocation as `this._httpClient.get<boolean>`